packages/react/src/components/Callout/Callout.types.ts

```diff
@@ -214,6 +214,14 @@ export interface ICalloutProps extends React.HTMLAttributes<HTMLDivElement>, Rea
    */
```

Comment:
## [🕵 fluentuiv8 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=29118&commitId=bb96c610278f39aaace0cd88e508434367cafc72&env=prod&clientType=FLUENTUI)


  <details><summary><b>ContextualMenu</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|ContextualMenu.With submenu - RTL.hover.chromium.png|1000| Changed|
|ContextualMenu.With submenu - RTL.click.chromium.png|1000| Changed|

</details>


  <details><summary><b>Pivot - Overflow</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Pivot - Overflow.Tabs - RTL.Narrow - Overflow menu.chromium.png|4448| Changed|

</details>


  <details><summary><b>TeachingBubble</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|TeachingBubble.WideIllustration.default.chromium.png|581| Changed|
|TeachingBubble.SmallHeadline.default.chromium.png|1074| Changed|
|TeachingBubble.WideIllustration - RTL.default.chromium.png|382| Changed|

</details>


  <details><summary><b>Tooltip</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Tooltip.Default.default.chromium.png|206| Changed|

</details>


  <details><summary><b>Tooltip - Multiple</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Tooltip - Multiple.Two Tooltips.hover outer.chromium.png|504| Changed|

</details>


  <details><summary><b>react-charting-AreaChart</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|react-charting-AreaChart.Basic.hover.chromium.png|3332| Changed|
|react-charting-AreaChart.Multiple - Dark Mode.hover.chromium.png|4522| Changed|
|react-charting-AreaChart.Basic - Dark Mode.hover.chromium.png|2864| Changed|
|react-charting-AreaChart.Multiple.hover.chromium.png|4208| Changed|

</details>


  <details><summary><b>react-charting-LineChart</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|react-charting-LineChart.Basic - Dark Mode.hover.chromium.png|3173| Changed|
|react-charting-LineChart.Basic.hover.chromium.png|2629| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>